roles/fstat/tasks/main.yml

```diff
@@ -31,3 +31,15 @@
     name: fstat
     enabled: True
     state: started
+
+- name: Configure Cron tab centos
+  cron:
+    name: fstat {{item.name}}
```

Comment:
@mscherer it won't complain, I have tested it in a local vm and it works just fine.